src/lib/hooks/organization/useGetOrganizations.tsx

```diff
@@ -29,15 +29,19 @@ export const useGetOrganizations = (realm?: string, userStorage?: string) => {
 
 	const execute = async (
 		realm: string,
-		{ identifiant, mail }: searchRequestOrganization,
+		{ identifiant, mail, typeRecherche }: searchRequestOrganization,
```

Comment:
why not getting the entire searchRequestOrganization and then pass the entire searchRequestOrganization